src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -58,27 +67,80 @@ const getStyles = (theme: Theme) => ({
     },
 });
 
+const SearchSelection =
```

Comment:
This component is being abstracted to make the code more readable. We can discuss if we would like to move it in a separate file if you feel it doesn't belong here.

Reply:
@JoakimSM would you mind if I take a note and do this is a later PR to avoid conflicts?